using-mobile-extensions/adobe-media-analytics/media-api-reference.md

```diff
@@ -1,14 +1,14 @@
 # Media API reference
 
-## extensionVersion
+### extensionVersion
```

Comment:
Updated all create API to h2.

Reply:
I'm in favour of keeping them all as H2s as well. If you want to make it an H3, it needs a better H2 around it, since createTracker isn't really a subfunction of extensionVerison.